source/howto/CreateAHeatStack.rst

```diff
@@ -60,7 +60,7 @@ For example:
 
   heat_template_version: 2018-03-02
 
-Indicates that the Heat Template contains features which have been added and/or removed up to the Queens release of OpenStack.
+Indicates that the Heat Template contains features which have been added and/or removed up to the Train release of OpenStack.
```

Comment:
For Heat the template version is for Queens, not Train (as the date is associated with Queens release)